media/redcore/js/lib/flexslider/jquery.flexslider.js

```diff
@@ -394,9 +394,9 @@
           accDx = 0;
 
         if(!msGesture){
-            el.addEventListener('touchstart', onTouchStart, false);
 
             function onTouchStart(e) {
+            el.addEventListener('touchstart', onTouchStart, false);
```

Comment:
@hanshenrikchr did you check? I think I have upgraded flexslider several times since this PR :)